bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/SearchResultServlet.java

```diff
@@ -216,6 +217,10 @@ private Resource getSearchContentResource(SlingHttpServletRequest request, Page
         return results;
     }
 
+    private String getId(Resource resource) {
+        return "search-" + StringUtils.substring(DigestUtils.sha1Hex(resource.getPath()), 0, 10);
```

Comment:
see comment above

Reply:
`PageListItemImpl` used for adding items to the list of results requires a `parentId` which in this case would be generated by the SearchResultServlet